standard/unsafe-code.md

```diff
@@ -1025,7 +1025,7 @@ stackalloc_initializer
 
 The *unmanaged_type* ([§8.8](types.md#88-unmanaged-types)) indicates the type of the items that will be stored in the newly allocated location, and the *expression* indicates the number of these items. Taken together, these specify the required allocation size. Since the size of a stack allocation cannot be negative, it is a compile-time error to specify the number of items as a *constant_expression* that evaluates to a negative value.
 
-A stack allocation initializer of the form stackalloc `T[E]` requires `T` to be an unmanaged type ([§22.3](unsafe-code.md#223-pointer-types)) and `E` to be an expression implicitly convertible to type `int`. The construct allocates `E * sizeof(T)` bytes from the call stack and returns a pointer, of type `T*`, to the newly allocated block. If `E` is a negative value, then the behavior is undefined. If `E` is zero, then no allocation is made, and the pointer returned is implementation-defined. If there is not enough memory available to allocate a block of the given size, a `System.StackOverflowException` is thrown.
+A stack allocation initializer of the form stackalloc `T[E]` requires `T` to be an unmanaged type (§8.8) and `E` to be an expression implicitly convertible to type `int`. The construct allocates `E * sizeof(T)` bytes from the call stack and returns a pointer, of type `T*`, to the newly allocated block. If `E` is a negative value, then the behavior is undefined. If `E` is zero, then no allocation is made, and the pointer returned is implementation-defined. If there is not enough memory available to allocate a block of the given size, a `System.StackOverflowException` is thrown.
```

Comment:
The text link says "unmanaged types", but the target is "pointer types". From both the context and the description in this paragraph, I think "unmanaged types" is a better target. However, this link has been to "pointer types" since C# 4.0 (or older). This one might be better left as is.

Reply:
I favor the link going to "unmanaged type," as that's what the text clearly says. Later, the para says "returns a pointer, of type `T*`, to ...", but until then no pointer is involved.

Reply:
@BillWagner Unless I'm missing something, all your replacement section numbers appear to hard-coded rather than being actual links of the form "[...](...)".